.gitignore

```diff
@@ -5,7 +5,7 @@ src/bitcoin
 src/zcashd
 src/zcash-cli
 src/zcash-gtest
-src/bitcoin-tx
+src/zcash-tx
 src/test/test_bitcoin
```

Comment:
The CI server *would* need to be updated, because it references the coverage directories which are named after the binaries. So I'm inclined to not change `test_bitcoin` for this PR.